xline/src/lib.rs

```diff
@@ -127,3 +127,5 @@ pub mod server;
 mod state;
 /// Storage module
 mod storage;
+
+pub use storage::storage_abstraction::Memory;
```

Comment:
Shall we make a naming convention about interface? I use `XxxInterface` in Curp(now I find it too tedious). In xline, there is `xxx_abstraction` and `LeaseTrait`. I suggest we name all interfaces to `XxxApi`.

Reply:
good suggestion, XxxApi is a better choice